week9/Week9/View/ContentView.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  ContentView.swift
+//  Week9
+//
+//  Created by Curtis DeGidio on 10/28/23.
+//
+
+import SwiftUI
+
+struct ContentView: View {
+    @State var searchText = ""
+    @State var confirmedSearchText: String?
+    @State var disableSearchField = false
+    @State var photos = [Photo]()
+    @ObservedObject var viewModel = SearchResults()
```

Comment:
I would consider using @StateObject here. @ObservedObject with a direct initialization like this is typically not recommended because you run the risk of it leading to unexpected behavior and potentially loss of data, especially if the view is re-rendered. 

In this case it works perfectly, but when in doubt, throw it in a @StateObject for the view responsible for the data, then use @ObservedObject in the child views where you can directly pass down the source of truth from the parent view.